data/json/itemgroups/clothing.json

```diff
@@ -397,5 +397,19 @@
       { "item": "socks" },
       { "item": "boots" }
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "slaughter_clothes",
```

Comment:
This new item group doesn't seem to be used anywhere.

Reply:
Removed and slated for another PR